src/profile/script.js

```diff
@@ -0,0 +1,268 @@
+  });
+
+  const userName = document.createElement('p');
+  userName.classList.add('userNameContainer');
+  cardContainer.appendChild(userName);
+  userName.innerText = `Username: ${username}`;
+
+  const dataContainer = document.createElement('div');
+  dataContainer.classList.add('dataContainer');
+  cardContainer.appendChild(dataContainer);
+
+  const dataInnerContainer = document.createElement('div');
+  dataInnerContainer.classList.add('dataInnerContainer');
+  dataContainer.appendChild(dataInnerContainer);
+
+  const oldDataContainer = document.createElement('div');
+  oldDataContainer.classList.add('oldDataContainer');
+  dataInnerContainer.appendChild(oldDataContainer);
+
+  const oldDataHeading = document.createElement('h3');
+  oldDataHeading.innerText = 'Old Data';
+  oldDataContainer.appendChild(oldDataHeading);
+
+  const oldUserInfoList = document.createElement('ul');
+  oldUserInfoList.classList.add('userInfoListContainer');
+  oldDataContainer.appendChild(oldUserInfoList);
+
+  //looping through the old data to display in list
+  for (const listItem in oldData) {
+    const li = document.createElement('li');
+    li.innerText = `${formatPropertyField(listItem)}: ${
+      oldData[listItem] || (listItem === 'yoe' ? NaN : '')
+    }`;
+    oldUserInfoList.appendChild(li);
+  }
+
+  const newDataContainer = document.createElement('div');
+  newDataContainer.classList.add('newDataContainer');
+  dataInnerContainer.appendChild(newDataContainer);
+
+  const newDataHeading = document.createElement('h3');
+  newDataHeading.innerText = 'New Data';
+  newDataContainer.appendChild(newDataHeading);
+
+  const newUserInfoList = document.createElement('ul');
+  newUserInfoList.classList.add('userInfoListContainer');
+  newDataContainer.appendChild(newUserInfoList);
+
+  //looping through the new data to display in list
+  for (const listItem in newData) {
+    const li = document.createElement('li');
+    li.innerText = `${formatPropertyField(listItem)}: ${
+      newData[listItem] || (listItem === 'yoe' ? NaN : '')
+    }`;
+    newUserInfoList.appendChild(li);
+  }
+
+  const buttonsContainer = document.createElement('div');
+  buttonsContainer.classList.add('buttonsContainer');
+  dataContainer.appendChild(buttonsContainer);
+
+  const approveBtn = document.createElement('button');
+  const rejectBtn = document.createElement('button');
+  approveBtn.innerText = 'Approve';
+  rejectBtn.innerText = 'Reject';
+
+  approveBtn.onclick = async () => {
+    document.getElementById('cover-spin').style.display = 'block';
+    try {
+      const response = await fetch(`${API_BASE_URL}/users/${username}`, {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+        body: JSON.stringify({
+          id: profileDiffId,
+          first_name: newData.first_name || '',
+          last_name: newData.last_name || '',
+          /* email: newData.email || '',
```

Comment:
Any reason to keep these comments?

Reply:
Removed.

Reply:
Earlier, we were adding those details of users too but then @lakshayman suggested not to add those details for now. Removed the comments for the time being.

Reply:
Thank you